scripts/dev/upstream_merge/kernel_build_conf.json

```diff
@@ -0,0 +1,22 @@
+{
+  "email_from": "dkaushik@emerson.com",
+  "email_to": "dkaushik@emerson.com",
+  "kernel_build": {
+    "kernel_src_dir": "nilrt-kernel-build/linux",
+    "target_host": "10.152.8.54",
+    "target_user": "admin",
+    "arch": "x86_64",
+    "temp_modules_dir": "nilrt-kernel-build/linux/tmp-glibc/modules",
```

Comment:
Shouldn't this just be `./tmp-modules` according to the [README](https://github.com/ni/nilrt/blob/nilrt/master/scarthgap/docs/README.kernel.md)? You should be able to build the path with the kernel source directory in your code, instead of in your config.